tests/unit/test_imitation_dynamics.py

```diff
@@ -0,0 +1,77 @@
+from nashpy.egt.imitation_dynamics import imitation_dynamics
+import numpy as np
+import unittest
+import random
+
+
+class TestImitationDynamics(unittest.TestCase):
```

Comment:
Can you modify all these to be just functional pytest calls? (There's no need for the Unittest class).

Take a look at https://github.com/drvinceknight/Nashpy/blob/main/tests/unit/test_fictitious_play.py#L27 for an example.

Reply:
Removed the unittest class as described